txpool/tx_object_test.go

```diff
@@ -29,6 +29,13 @@ func newChainRepo(db *muxdb.MuxDB) *chain.Repository {
 	return repo
 }
 
+func newChainRepoCustomTimestamp(db *muxdb.MuxDB, timestamp uint64) *chain.Repository {
+	gene := genesis.NewDevnetCustomTimestamp(timestamp)
```

Comment:
What's the purpose of new genesis with timestamp? Is it for the following code? https://github.com/vechain/thor/blob/352fbc6f4e1faf442e6efff255307b427093bc86/txpool/tx_pool.go#L238

Reply:
I don't remember the exact path of the code but its probably that one. The purpose was to get more coverage in areas that I could not cover otherwise with the default timestamp.

Reply:
I would suggest two approaches:
1. Use the genesis builder to build a minimal genesis, like [this](https://github.com/vechain/thor/blob/master/consensus/consensus_test.go#L56)
2. Add a new block with the current time, add it to the repository, and set it with best would make loops inside txpool start working.

Reply:
@MakisChristou  Another approach https://github.com/vechain/thor/blob/master/genesis/customnet.go#L35, but I like to use the genesis builder for testing purposes.